frontend/subscriptions/addressBook.js

```diff
@@ -82,4 +85,16 @@ export default (subscribe) => {
     addressClone.tags.push(defTag);
     dispatch(updateAddress(addressClone, true));
   });
+
+  // Dispatch action to backend to delete the given addresses after successful confirmation
+  subscribe(userAddressesDelete$, ({ dispatch, action }) => {
+    const { addressIds } = action;
+
+    dispatch(showModal({
+      confirm: 'address.delete.button',
+      dismiss: 'modal.dismiss',
+      title: 'address.delete.confirmationDialog.title',
+      message: 'address.delete.confirmationDialog.message',
+    })).then(confirmed => confirmed && dispatch(deleteAddresses(addressIds)));
```

Comment:
At the moment the content of address book jumps because it immediately jumps back to the overview page and then after some time (when deleteUser is done) the page rerenders without the deleted address.

I'd suggest to either stay on the details page and show loading bar (and after the new getUser is done) redirect, or at do it the way it is at the moment and add a loadingbar to the overview.